django_project/certification/templates/course/create.html

```diff
@@ -62,4 +67,120 @@ <h1>Add Course</h1>
+            <form method="post" id="preview-form" action="/en/{{ project_slug }}/certifyingorganisation/{{ organisation_slug }}/preview-certificate/" target="_blank" onsubmit="return previewSubmit()">
+                {% csrf_token %}
+                <input type="hidden" name="course_type" value="">
+                <input type="hidden" name="course_convener" value="">
+                <input type="hidden" name="training_center" value="">
+                <input type="hidden" name="start_date" value="">
+                <input type="hidden" name="end_date" value="">
+                <input type="hidden" name="template_certificate" value="">
+                <input id="preview-certificate" type="submit" class="btn btn-default" value="Preview Certificate" form="preview-form">
+            </form>
+            <p id="error-submit" style="color: darkred; font-weight: bold"></p>
+        </div>
+    </section>
+
+    <script>
+        $(document).ready(function () {
+            $('#preview-certificate').appendTo($('#div_id_template_certificate'));
+            $('#error-submit').appendTo($('#div_id_template_certificate'));
+            $('#preview-template-load').appendTo($('#div_id_template_certificate'));
+        });
+
+        function previewSubmit() {
+            $('#error-submit').html('');
+
+            if($('select[name=course_type]').val() === ''){
+                $('#error-submit').html('Please choose a course type.');
+                return false
+            }else if($('select[name=course_convener]').val() === ''){
+                $('#error-submit').html('Please choose a course convener.');
+                return false
+            }else if($('select[name=training_center]').val() === ''){
+                $('#error-submit').html('Please choose a training center.');
+                return false
+            }else if($('input[id=id_start_date]').val() === ''){
+                $('#error-submit').html('Please choose start date.');
+                return false
+            }else if($('input[id=id_end_date]').val() === ''){
+                $('#error-submit').html('Please choose end date.');
+                return false
+            }
+
+            $('#preview-form input[name=course_convener]').val($('select[name=course_convener]').val());
+            $('#preview-form input[name=course_type]').val($('select[name=course_type]').val());
+            $('#preview-form input[name=training_center]').val($('select[name=training_center]').val());
+            $('#preview-form input[name=start_date]').val($('input[id=id_start_date]').val());
+            $('#preview-form input[name=end_date]').val($('input[id=id_end_date]').val());
+        }
+
+        //check if browser supports file api and filereader features
+        if (window.File && window.FileReader && window.FileList && window.Blob) {
+
+           //this is not completely neccesary, just a nice function I found to make the file size format friendlier
```

Comment:
__[Misspell]__

"neccesary" is a misspelling of "necessary"


<!-- Comment by [SideCI](https://sideci.com) -->